### DISCUSSION TITLE
cloudflare解析不了根域名

### DISCUSSION THREAD

**Discussion by Coffee-with-salt** [jeessy2/ddns-go#1080] [Q&A]
<img width="419" alt="微信图片_20240417123713" src="https://github.com/jeessy2/ddns-go/assets/137048660/82294e6b-4b58-4037-8b61-cbe36953f70c">
版本是V6.3.2用的dns服务商是cloudflare  总会报错：在DNS服务商中未找到域名，这个域名是根域名。
肯定在DNS服务商里面写了

**Comment by jeessy2** [comment]
经测可以解析。确定填对了？

**Reply by Coffee-with-salt** [reply]
token的问题 更换域名后 没有更新cloudflare的token，更新为全区域口令后正常了 感谢回复
